### PR TITLE
unit tests for StringMethods

### PR DESCRIPTION
**Pull request by jwojciec**
Proposal of unit tests. Started with simple unit test examples for StringMethods.java

### CODE REVIEW

File: app/src/test/java/com/notecrypt/utils/StringMethodsTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class StringMethodsTest {
+
+    private final static String TEST_PATH = "test_path";
```
Comment (jwojciec) [inline app/src/test/java/com/notecrypt/utils/StringMethodsTest.java]: probably not the best example

File: app/build.gradle [hunk 2]
```diff
@@ -23,4 +23,6 @@ dependencies {
     implementation 'com.android.support:support-compat:28.0.0-rc01'
     implementation 'com.android.support:appcompat-v7:28.0.0-rc01'
     implementation 'com.android.support:design:28.0.0-rc01'
+
+    testCompile 'junit:junit:4.12'
```
Comment (RyuzakiKK) [inline app/build.gradle]: Change it to `testImplementation` because `Compile` is deprecated.

**Review by RyuzakiKK — CHANGES_REQUESTED** [review]
Apart from replacing `testCompile`, this PR is very good. Thank you.
When you'll fix it I'll do a merge.

File: app/build.gradle (hunk 2 above)
Comment (jwojciec) [reply app/build.gradle]: 👍 fixing that, thx.

**Review by RyuzakiKK — APPROVED** [review]

**Comment by RyuzakiKK** [comment]
LGTM, thanks!

**Pull request merged**